### PR TITLE
Update index.html.slim

### PR DESCRIPTION
**Pull request by brocked**
Redirecting Promo2 to new devnation-pass-giveaway minipage

### CODE REVIEW

**Comment by adelasofia** [comment]
this is ok to test

File: index.html.slim [hunk 1]
```diff
@@ -15,7 +15,7 @@ description: The simple, modern & productive way to build apps and infrastructur
         a.spotlight-item href='http://developers.redhat.com/promotions/devnation-discount-code'
           img(src="http://static.jboss.org/images/rhd/promo/RHDev_homepromo_devnation_30percent_18apr2016.png" alt="Get 30% off registration for DevNation")
           span Join RHD and get a special 30% discount code.
-        a.spotlight-item href='http://developers.redhat.com/register'
+        a.spotlight-item href='http://developers.redhat.com/promotions/devnation-pass-giveaway/'
```
Comment (LightGuard) [inline index.html.slim]: All the links should be in the form of `#{site.base_url}/path/to/resource` so they go to the correct location based on the environment.

File: index.html.slim [hunk 2]
```diff
@@ -15,7 +15,7 @@ description: The simple, modern & productive way to build apps and infrastructur
         a.spotlight-item href='http://developers.redhat.com/promotions/devnation-discount-code'
           img(src="http://static.jboss.org/images/rhd/promo/RHDev_homepromo_devnation_30percent_18apr2016.png" alt="Get 30% off registration for DevNation")
           span Join RHD and get a special 30% discount code.
-        a.spotlight-item href='http://developers.redhat.com/promotions/devnation-pass-giveaway/'
+        a.spotlight-item href='#{site.base_url}promotions/devnation-pass-giveaway'
```
Comment (LightGuard) [inline index.html.slim]: You're missing the `/` between `}` and `promotions`

Comment (brocked) [inline index.html.slim]: Dang. Fixed it. Noted for future links.

**Comment by LightGuard** [comment]
@brocked Great, thanks! Looks good now, let's verify with the tests.

**Comment by adelasofia** [comment]
retest this please

**Pull request merged**